llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -4188,6 +4188,133 @@ define void @foo(i32 %arg) {
   EXPECT_EQ(NewPHI->getNumIncomingValues(), PHI->getNumIncomingValues());
 }
 
+void checkSwapOperands(sandboxir::Context &Ctx, llvm::sandboxir::CmpInst *SBCmp,
```

Comment:
Shouldn't this be `static void checkSwapOperands(...)` ?

Also `llvm::sandboxir::CmpInst` -> `sandboxir::CmpInst`

Also we are trying to move away from the `SB` prefix. We are still using the `LLVM` prefix for the `llvm::` values though so you can tell them apart.